### PR TITLE
Fix weird url issue with templates

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: base/templates/base.html [hunk 1]
```diff
@@ -18,8 +18,7 @@
                     </form>
                 </div>
             {% else %}
-                <!-- TODO broken for some reason. should use url with reverse -->
-                <a class="navbar-brand" href="/">MySpotifyTool</a>
+                <a class="navbar-brand" href="{% url 'base:index' %}">MySpotifyTool</a>
```
Comment (jlward) [inline base/templates/base.html:21]: I don't think I actually fixed it. But I at least added a test case to show if it's happening. We should get you to checkout this branch and see if it's busted. If it is, I have a few things we can try. If it isn't, then we can just not merge this in.

File: base/templates/base.html [hunk 2]
```diff
@@ -18,8 +18,7 @@
                     </form>
                 </div>
             {% else %}
-                <!-- TODO broken for some reason. should use url with reverse -->
```
Comment (jlward) [inline base/templates/base.html:21]: Any reason you switched from django template comment to an HTML comment? This one will actually show up in the page. Where a Django template comment will not.

File: libraries/templates/compare_stats.html [hunk 3]
```diff
@@ -23,7 +23,7 @@ <h1 style="padding-bottom:35px">Compare Library Stats</h1>
         <tr id="username">
           <th scope="col"></th>
           <th scope="col" class="field">{{ my_library.user.username }}</th>
-          <th scope="col" class="field>Difference</th>
+          <th scope="col" class="field">Difference</th>
```
Comment (jlward) [inline libraries/templates/compare_stats.html:26]: Someone did a bad thing. We can talk about how to mitigate something like this in the future.

File: libraries/tests/test_views.py [hunk 4]
```diff
@@ -157,7 +157,7 @@ def test_GET_returns_200(self):
     def test_user_name_present(self):
         r = self.client.get(self.url_compare)
         actual = self.css_select_get_text(r, 'tr#username th')
-        expected = ['', self.user.username, self.other_user.username]
+        expected = ['', self.user.username, 'Difference', self.other_user.username]
```
Comment (jlward) [inline libraries/tests/test_views.py:160]: This test needed to be updated once the HTML fix happened.

File: base/templates/base.html (hunk 2 above)
Comment (derekjamerson) [reply base/templates/base.html:21]: Oh, I didn't realize. I just used the shortcut key and it did it that way, so I made the others match.

File: libraries/tests/test_views.py (hunk 4 above)
Comment (derekjamerson) [reply libraries/tests/test_views.py:160]: I was so puzzled why that item was missing from the list! Smh at myself.

**Comment by jlward** [comment]
I was not able to reproduce the issue on my machine, but it could be browser specific (I was using chrome). My suspicion is that somewhere in the HTML there is some malformed tags somewhere. Either a missing `"` or a missing closing tag. I was hoping my editor would find it for me, but it did not. If that's is what is happening, we can talk about using something like [hamlp](https://pypi.org/project/django-hamlpy/) to deal with it while also teaching you about custom template loaders. Alternatively, we could do something like making all GET requests auto check to see if the HTML returned is valid. Some combination of these approaches will likely eventually sort out what is going on.

**Pull request merged**